FireRender.Maya.Src/Context/TahoeContext.cpp

```diff
@@ -252,6 +253,18 @@ void TahoeContext::setupContext(const FireRenderGlobalsData& fireRenderGlobalsDa
 
 	// OCIO
 	{
+		const auto& eVars = EnvironmentVarsWrapper<char>::GetEnvVarsTable();
+		auto envOCIOPath = eVars.find("OCIO");
```

Comment:
could we get rid of "auto" ?

Reply:
envOCIOPath is ::const_iterator
Haven't we decided to use auto for iterators?